client/src/components/cart/CartTable.js

```diff
@@ -0,0 +1,187 @@
+import { useContext } from "react";
+import styled from "styled-components";
+import Payment from "../forms/Payment";
+import { Link } from "react-router-dom";
+import { CartContext } from "../context/Context";
+function CartTable() {
+  const [cart, setCart] = useContext(CartContext);
+  const toggleItemQuantity = (itemName, subtract) => {
+    let newState;
+    if (subtract) {
+      newState = cart.map((item) => {
+        item.name === itemName && item.quantity--;
+        return item;
+      });
+    } else {
+      newState = cart.map((item) => {
+        item.name === itemName && item.quantity++;
+        return item;
+      });
+    }
+    setCart(newState);
+  };
+  const handleDelete = (name) => {
+    const filteredCart = cart.filter((item) => item.name != name);
+    setCart(filteredCart);
+  };
+  return (
+    <TableContainer>
+      <h2>Your Cart</h2>
+      {cart.length ? (
+        <>
+          <table>
+            <thead>
+              <tr>
+                <th>Product</th>
+                <th>Price</th>
+                <th>Quantity</th>
+                <th>Total</th>
+              </tr>
+            </thead>
+            <tbody>
+              {cart.map((item, index) => (
+                <tr key={index}>
+                  <td className="flex">
+                    <img src={item.imageUrl} alt="" />
```

Comment:
Remember to set an `alt` for accessibility. It's helpful for people using screen readers.

Reply:
Yea, I should do that more.